src/main/webapp/components/Documents.js

```diff
@@ -54,7 +66,21 @@ export const Documents = hh(class Documents extends Component {
     this.setState({ showAddAdditionalDocuments: !this.state.showAddAdditionalDocuments });
   };
 
+  findDul = () => {
+    let dulPresent = false;
+    if (this.props.keyDocuments.length !== 0) {
+      this.props.keyDocuments.forEach(docs => {
+        if (docs.fileType === 'Data Use Letter') {
```

Comment:
It would be nice if `'Data Use Letter'` were an enum so we could abstract this kind of check. Could easily be a tech-debt story if you think it's a good idea.
https://stackoverflow.com/questions/287903/what-is-the-preferred-syntax-for-defining-enums-in-javascript